api/data/info.go

```diff
@@ -13,6 +13,10 @@ const (
 	bktSettingsObject                  = ".s3-settings"
 	bktCORSConfigurationObject         = ".s3-cors"
 	bktNotificationConfigurationObject = ".s3-notifications"
+
+	VerUnversioned = "Unversioned"
```

Comment:
It seems we can use empty string here or drop this constant at all (since `Unversioned` string isn't used as I understand). So when we create new BucketSettings like:
```go 
bktSettings := &BucketSettings{}
```
we get settings with "Unversioned" status if we update the following function:
```go
func (b BucketSettings) Unversioned() bool {
	return len(b.Versioning) == 0
}
```

Reply:
You're right, but constant string `Unversioned` makes code more readable (someone who will read the code will not need know what does empty string mean). I'm not sure we should replace it by empty string or drop it at all.